src/helper.js

```diff
@@ -983,6 +985,40 @@ export function parseItemTypeFromLore(lore) {
   };
 }
 
+export function getCacheFilePath(dirPath, type, name) {
+  // we don't care about folder optimalization when we're developing
+  if (process.env?.NODE_ENV == "development") {
+    return path.resolve(dirPath, `${type}_${name}.png`);
+  }
+
+  let subdirs = [type];
```

Comment:
const?

Reply:
not sure, since i potentially push into it, but i'll try.

Reply:
Yeah pushing is not an issue, only thing you can't do is reassign the value completely